torchreid/apis/export.py

```diff
@@ -21,13 +21,13 @@
 
 from torch.onnx.symbolic_registry import register_op
 
-from scripts.script_utils import group_norm_symbolic
+from scripts.script_utils import group_norm_symbolic, patch_InplaceAbn_forward
```

Comment:
Please revert all the changes in this file. Public API is used by 3rd party entities, so changing it requires changes in all API clients. I'll rename inputs/outputs later.